cmd/frontman/main.go

```diff
@@ -365,6 +383,99 @@ func handleFlagDaemonizeMode(daemonizeMode bool) {
 	}
 }
 
+func handleServiceCommand(ca *frontman.Frontman, check, start, stop, restart bool) {
+	if !check && !start && !stop && !restart {
+		return
+	}
+
+	svc, err := getServiceFromFlags(ca, "", "")
+	if err != nil {
+		log.WithError(err).Fatalln("can't find service")
+	}
+
+	var status service.Status
+	if status, err = svc.Status(); err != nil {
+		log.WithError(err).Fatalln("can't get service status")
+	}
+
+	if check {
+		switch status {
+		case service.StatusRunning:
+			fmt.Println("running")
+		case service.StatusStopped:
+			fmt.Println("stopped")
+		case service.StatusUnknown:
+			fmt.Println("unknown")
+		}
+
+		os.Exit(0)
```

Comment:
I think this function can be simplified by just putting a os.Exit(0) at end of function, and use else-if blocks to control execution where needed. Only exit early if you have error code 1.

Reply:
I think the current code is OK too, given that it was existing before this PR.